Core/OxidEE_Events.php

```diff
@@ -153,9 +153,8 @@ private static function _createOrderTransactionTable()
             `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
             `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
             `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
-            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
-            `WDOXIDEE_STATE` enum('success','error','pending'),
-            `WDOXIDEE_PAYMENTMETHOD` varchar(32) NOT NULL,
+            `WDOXIDEE_ACTION` enum('authorization','purchase') NOT NULL DEFAULT 'purchase',
```

Comment:
As we have many(more then 10) of payment actions. Is it maybe better to just have a varchar?

Reply:
We have added those other actions in a separate `TYPE column` in WDPD-24. The 2 actions that are shown here are just referring to the **payment action** configured by the admin.

<img width="257" alt="Screenshot 2019-03-28 at 17 38 21" src="https://user-images.githubusercontent.com/12810026/55175836-57126d80-5180-11e9-9ec6-522b4ee0f07a.png">